### PR TITLE
Comments database

### LINKED ISSUE #40: comment db migration

### PR DESCRIPTION
**Pull request by rsmith88**
Fixes #40

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```
Comment (toddpla) [inline app/models/comment.rb]: 👍

File: spec/factories/comments.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# FactoryGirl.define do
+#   factory :comment do
```
Comment (toddpla) [inline spec/factories/comments.rb]: can we remove these comments

File: spec/models/comment_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  # User.new(name: 'Jeff', email: 'jeff@hotmail.com', password: 'password1')
```
Comment (toddpla) [inline spec/models/comment_spec.rb]: can we get rid of the comments please

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```
Comment (imogenmisso) [inline app/models/comment.rb]: Nice use of foreign keys

File: spec/models/comment_spec.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  # Post.new(message: 'This is a post', user_id: 1)
+  #
+
+  subject do
+    user = FactoryGirl.create(:user)
+    post = FactoryGirl.create(:post, user: user)
+    described_class.new(comment: "This is a test", user_id: 1, post_id: 1)
```
Comment (toddpla) [inline spec/models/comment_spec.rb]:
```suggestion
    described_class.new(comment: "This is a test", user: user, post: post)
```
can you try this and see if this works - i think the more rails way of doing things

File: app/models/comment.rb (hunk 4 above)
Comment (runticle) [reply app/models/comment.rb]: i dont know, do we?

File: spec/factories/comments.rb (hunk 2 above)
Comment (runticle) [reply spec/factories/comments.rb]: ok todd!

File: spec/models/comment_spec.rb [hunk 6]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  # User.new(name: 'Jeff', email: 'jeff@hotmail.com', password: 'password1')
+  # Post.new(message: 'This is a post', user_id: 1)
+  #
+
+  subject do
+    user = FactoryGirl.create(:user)
```
Comment (imogenmisso) [inline spec/models/comment_spec.rb]: Might need to change all these to FactoryBot before we merge

File: app/models/comment.rb (hunk 4 above)
Comment (runticle) [reply app/models/comment.rb]: we did

File: app/models/comment.rb (hunk 1 above)
Comment (runticle) [reply app/models/comment.rb]: thanks todd

File: spec/models/comment_spec.rb (hunk 3 above)
Comment (runticle) [reply spec/models/comment_spec.rb]: done

File: spec/models/comment_spec.rb (hunk 5 above)
Comment (runticle) [reply spec/models/comment_spec.rb]: done!

Comment (runticle) [reply spec/models/comment_spec.rb]: it works!

**Comment by runticle** [comment]
⭐️

**Comment by toddpla** [comment]
⭐️

File: app/models/comment.rb (hunk 4 above)
Comment (toddpla) [reply app/models/comment.rb]: nice one 👍

File: app/models/comment.rb (hunk 1 above)
Comment (toddpla) [reply app/models/comment.rb]: no problem

**Pull request merged**